### PR TITLE
Regex filtering in logEntryEventStream component

### PR DESCRIPTION
**Pull request by jamessimone**
* Fixes (some) of #212 by adding regular expression filtering to the `logEntryEventStream` component. Did some cleanup, as well

### CODE REVIEW

File: nebula-logger/main/log-management/lwc/logEntryEventStream/logEntryEventStream.js [hunk 1]
```diff
@@ -104,44 +97,38 @@ export default class LogEntryEventStream extends LightningElement {
 
     onToggleStream() {
         this.isStreamEnabled = !this.isStreamEnabled;
-        if (this.isStreamEnabled) {
-            this.createSubscription();
-        } else {
-            this.cancelSubscription();
-        }
+        // eslint-disable-next-line
+        this.isStreamEnabled ? this.createSubscription() : this.cancelSubscription();
```
Comment (jamessimone) [inline nebula-logger/main/log-management/lwc/logEntryEventStream/logEntryEventStream.js:101]: let me know what you think - when I wrote this, I thought it was nice, but then eslint hated it so ...

Comment (jongpie) [reply nebula-logger/main/log-management/lwc/logEntryEventStream/logEntryEventStream.js:101]: Yeah, I like this approach - I also think it looks nice, so let's stick with this.

File: nebula-logger/main/log-management/lwc/logEntryEventStream/logEntryEventStream.js [hunk 2]
```diff
@@ -68,24 +75,9 @@ export default class LogEntryEventStream extends LightningElement {
...
-        this.originLocationFilter = event.target.value;
-    }
-
-    handleLoggedByFilterChange(event) {
-        this.loggedByFilter = event.target.value;
-    }
-
-    handleMessageFilterChange(event) {
-        this.messageFilter = event.target.value;
+    handleFilterChange(event) {
```
Comment (jongpie) [inline nebula-logger/main/log-management/lwc/logEntryEventStream/logEntryEventStream.js:78]: Awww yeah, I love this change - having only 1 common `change` function is much better 🚀

**Review by jongpie — APPROVED** [review]
This looks awesome - thanks again for making these changes!

**Pull request merged**